modules/data.atmosphere/R/download.CRUNCEP_Global.R

```diff
@@ -1,30 +1,39 @@
-##' Download and conver to CF CRUNCEP single grid point from MSTIMIP server using OPENDAP interface
```

Comment:
This is the deleted line, which I fixed below. I abbreviated the formal title of each function to just the download part, and maintained the subsequent line (the description field) as the full, original description.